Source/IGListAdapterUpdater.m

```diff
@@ -84,6 +84,7 @@ - (void)performReloadDataWithCollectionView:(UICollectionView *)collectionView {
 
     [delegate listAdapterUpdater:self willReloadDataWithCollectionView:collectionView];
     [collectionView reloadData];
+    [collectionView.collectionViewLayout invalidateLayout];
     [collectionView layoutIfNeeded];
```

Comment:
Will check, I'll need to spin through the places in Instagram.app that we call `reloadDataWithCompletion:` to make sure there are no weird side effects.